es6-destructuring/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <script>
+    const book1 = {
+      title: 'Goodnight Punpun',
+      author: 'Inio Asano',
+      libraryID: 3353
+    };
+
+    const { title } = book1;
+    const { author } = book1;
+    const { libraryID } = book1;
```

Comment:
This works,  but idiomatic JS would put these all on one line. Do you know how to destructure more than one variable at a time? Try changing this to do that.